tailwind.config.ts

```diff
@@ -6,6 +6,16 @@ import { iconCollection } from './build/customIconCollection'
 export default {
   content: ['./index.html', './src/**/*.{vue,js,ts,jsx,tsx}'],
 
+  safelist: [
+    'icon-[lucide--folder]',
+    'icon-[lucide--package]',
+    'icon-[lucide--image]',
+    'icon-[lucide--video]',
+    'icon-[lucide--box]',
+    'icon-[lucide--audio-waveform]',
+    'icon-[lucide--message-circle]'
+  ],
+
```

Comment:
I don't think you'll need this if you use the pattern in https://github.com/Comfy-Org/ComfyUI_frontend/pull/5453

Reply:
You're right about the iconify plugin from PR #5453, but that approach only works for static template usage like:
```html
<i  class="icon-[lucide--folder]">
```

 The safelist is still needed for dynamic usage where icons are assigned as strings in JavaScript objects:
  ```js
{ icon: 'icon-[lucide--folder]' }  // Tailwind can't detect this
```

We have 3 icons used this way in `useAssetBrowser.ts` (we have to map them to api categories) and menu composables. However, 4 icons in the safelist are unused and can be removed.

The iconify plugin can't solve the dynamic string assignment problem. Tha'ts why we need the safelist.